### DISCUSSION TITLE
mapboxgl.accessToken is cleared after initialize DeckGL

### DISCUSSION THREAD

**Discussion by fifzteen** [visgl/deck.gl#5668] [Q&A] [answered]
I make a pen for an another Question. using the deck.gl scripting API.
After `new DeckGL`, mapboxgl.acccessToken is cleared.

https://codepen.io/fifzteen/pen/yLgKpbp

Enter your accessToken, and watch console.
From "create deck" accessToken is "no-token".
Is the way to init DeckGL without clearing accessToken exists?

**Comment by Pessimistress** [comment, marked as answer]
The `DeckGL` scripting interface creates a Mapbox map automatically: https://deck.gl/docs/api-reference/core/deckgl#properties

If you don't want to use this convenience feature then construct a `Deck` instance instead.

**Reply by fifzteen** [reply]
@Pessimistress 
Oh! sorry. I didn't check the document enough... Thank you.
